src/components/views/rooms/RoomPreviewBar.tsx

```diff
@@ -57,6 +57,9 @@ enum MessageCase {
 }
 
 interface IProps {
+    // The id of the room to be previewed
+    roomId?: string;
+
```

Comment:
I suggest you put this next to `room`. It is currently rather out-of-place.

Reply:
moved it to the `room`